pkg/runner/local_docker.go

```diff
@@ -307,7 +307,7 @@ func (r *LocalDockerRunner) Run(ctx context.Context, input *api.RunInput, ow io.
 		return nil, pretty.Wait()
```

Comment:
We should learn about the runID here too. For example:

```suggestion
              return &api.RunOutput{RunID: input.RunID}, pretty.Wait()
```